### PR TITLE
Dedupe

### LINKED ISSUE #162: Post-merge of exact same entities

### PR DESCRIPTION
**Pull request by joecummings**
Closes #162

### CODE REVIEW

**Review by elizlee — COMMENTED** [review]
One question; the rest looks good

File: cdse_covid/pegasus_pipeline/ingesters/ingest_aif.py [hunk 1]
```diff
@@ -20,14 +20,28 @@ def get_abstract_doc_id(graph: Graph) -> Optional[str]:
...
     if doc_id:
         doc_id_pair = tuple(doc_id.pop())
         return str(doc_id_pair[0].value) + ".ttl"
 
+    query = """
+    SELECT ?doc_id
+    WHERE {
+        ?e a aida:TextJustification .
+
+        ?e aida:source ?doc_id .
```
[the remark this answers is not in the document]
Comment (joecummings) [reply cdse_covid/pegasus_pipeline/ingesters/ingest_aif.py:36]: This  was actually a hack to get the AIF doc from our system and from UIUC's system. The better was to handle this is through our AIF generation. I will implement that and remove this code.

File: cdse_covid/pegasus_pipeline/ingesters/convert_claim_json.py [hunk 2]
```diff
@@ -686,6 +688,7 @@ def write_qnode_data(
             af.write("\t\taida:system <" + CDSE_SYSTEM + "> ] ;\n")
             af.write('\taida:endOffsetInclusive "' + str(end_offset_inclusive) + '"^^xsd:int ;\n')
             af.write('\taida:source "' + source + '"^^xsd:string ;\n')
+            af.write('\taida:sourceDocument "' + str(data["doc_id"]) + '"^^xsd:string ;\n')
```
Comment (elizlee) [inline cdse_covid/pegasus_pipeline/ingesters/convert_claim_json.py:691]: `source` and `sourceDocument` would be the same. Is that what we want in this case?

Comment (joecummings) [reply cdse_covid/pegasus_pipeline/ingesters/convert_claim_json.py:691]: For now, yes. In the merge process, once we have the correct filename, we update the graph with the correct doc.

Comment (elizlee) [reply cdse_covid/pegasus_pipeline/ingesters/convert_claim_json.py:691]: Cool, got it

**Review by elizlee — APPROVED** [review]

**Pull request merged**